http/src/main/java/org/sapia/ubik/rmi/server/transport/http/HttpConsts.java

```diff
@@ -60,4 +60,6 @@
    * @see #HTTP_CLIENT_MAX_CONNECTIONS_KEY
    */
   public static final int DEFAULT_MAX_CLIENT_CONNECTIONS = 25;
+  
+  public static final long DEFAULT_TIMED_OUT_CONNECTION_CLEANING_INTERVAL = 10000;
```

Comment:
Wonder if 10 seconds is not too long... I wonder if we should reduce that to to 5 or even 3 seconds. When nodes get unreachable (because leaving abruptly) it does create an increase in connection attempt. With a large cluster this could create contention... 10 secondes is quite long and a lot can happen in that time (relatively speaking).

Reply:
Yep could do that. Also can make that configurable. Gimme two secs ;)

Reply:
And I'll have a cold beer ;)